lib/MetaCPAN/Document/Favorite.pm

```diff
@@ -41,7 +41,7 @@ Sets the C<_timestamp> field to the value of L</date>.
 
 has timestamp => (
     is        => 'ro',
-    timestamp => { path => 'date', store => 1 },
+    timestamp => {}, # { path => 'date', store => 1 },
```

Comment:
Also to consider: `_timestamp` seems to be depracated in ES2+
See https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-timestamp-field.html